concent_api/core/tests/test_core_views.py

```diff
@@ -286,6 +286,40 @@ def test_send_reject_message_save_as_receive_out_of_band_status(self):
         self.assertEqual(reject_response.status_code, 202)
         self.assertEqual(Message.objects.last().type, message.RejectReportComputedTask.TYPE)
 
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_reject_message_when_timestamp_too_old(self):
+        response = self.client.post(
+            reverse('core:send'),
+            data = dump(
+                message.Ping(
+                    timestamp=int(dateutil.parser.parse("2017-11-17 09:40:00").timestamp())
```

Comment:
You're missing an `assert` here. You should assert that the difference between the current time and your timestamp is greater than `MSG_TTL` defined in golem-messages. Currently it is but if this ever gets changed our tests will silently just stop working.

Reply:
:+1: